tailwind.config.ts

```diff
@@ -183,6 +184,9 @@ export default {
       3: 3,
       4: 4,
     },
+    maxWidth: {
+      46: "11.5rem",
```

Comment:
What was the reason for this addition? Is it that size in Figma or it makes things look better?